azurerm/internal/services/media/parse/media_asset.go

```diff
@@ -0,0 +1,38 @@
+package parse
```

Comment:
minor but we have a generator that will auto generate these files now 🙂 idealy we should use that but its not a blocker